### DISCUSSION TITLE
Can modal labels only set confirm?

### DISCUSSION THREAD

**Discussion by Eve-Sama** [mantinedev/mantine#6397] [Q&A] [answered]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.10.1

### What package has an issue?

@mantine/modals

### What framework do you use?

create-react-app (CRA)

### In which browsers you can reproduce the issue?

Chrome

### Describe the bug

<img width="724" alt="图片" src="https://github.com/mantinedev/mantine/assets/30228406/75f31a71-90c4-46d6-8708-e25db2ffb1f7">

I just need `confirm` label. But if I only set `confirm`, it will lead to a error.

```tsx
      modals.openConfirmModal({
        title: 'test',
        children: (
          <>
            test
          </>
        ),
        // Error: Property 'cancel' is missing in type '{ confirm: string; }' but required in type 'ConfirmLabels'.
        labels: { confirm: 'Ok' },
        onConfirm: () => {
           // something
        },
      });
```

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
`labels: { confirm: 'Ok', cancel: '' }`

**Reply by Eve-Sama** [reply]
but ui is not right

```tsx
labels: { confirm: '好的', cancel: '' },
```

<img width="163" alt="图片" src="https://github.com/mantinedev/mantine/assets/30228406/15bd0ee6-6858-4dcf-bae9-08af766ccf2d">

**Reply by rtivital** [reply, marked as answer]
You can hide cancel button by setting display: none styles with `cancelProps` – https://mantine.dev/x/modals/#confirm-modal
